salesagility/SuiteCRM
# Consolidate global search settings (AOD, Basic)

## Description




- Removed a duplicate "Global Search" link in the admin panel.
- Moved AOD to "Search Settings" page.
- Moved SearchModules to "Search Settings" page. 
- Removed AOD enable option, this now comes from the search engine drop-down.
- Minor coding standard fixes, code simplification, cleanup, etc.
- Minor styling fixes, each of the search settings should be using standard panels:
- Cleaned up flaky ElasticSearch tests (These should be improved further / overhauled if we upgrade ElasticSearch versions).

![searchsettings2](https://user-images.githubusercontent.com/26431166/116036196-3ae2a280-a65e-11eb-8178-9135c0f1e8f7.png)


## Motivation and Context

- This is to consolidate and simplify the search options within the CRM. All settings should work exactly the same as before just now in one location.

## How To Test This

1. Check AOD works as expected.
3. Check basic search works as expected.
4. Check module selection works as expected.
5. Check that all settings visible on the page work and that nothing is missing.

## Types of changes


### Final checklist
